gulpfile.js

```diff
@@ -156,19 +164,12 @@ function generateManifest() {
 }
 
 gulp.task('watch-manifest', function() {
-  gulp.watch(
-    MANIFEST_SOURCE_FILES,
-    batch(function(events, done) {
-      endOfStream(generateManifest(), function() {
-        done();
-      });
-    })
-  );
+  gulp.watch(MANIFEST_SOURCE_FILES, generateManifest);
```

Comment:
This is a simplification of the above and fix for recent Node versions. See commit message for notes.